dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1024,6 +1030,40 @@ LanguageClientInterface convertCLIStringToEnum(String descriptor) {
         throw new UnsupportedOperationException("language not supported yet");
     }
 
+    /**
+     * Loads docker images from file system if there are any
+     */
+    public void loadDockerImages() {
+        INIConfiguration config = Utilities.parseConfig(this.getConfigFile());
+        String dockerImageDirectory = config.getString("docker-images");
+        if (!StringUtils.isBlank(dockerImageDirectory)) {
+            Path directoryPath = Paths.get(dockerImageDirectory);
+            Supplier<Stream<Path>> list = () -> {
+                try {
+                    return java.nio.file.Files.list(directoryPath);
+                } catch (NotDirectoryException e) {
+                    System.out.println("Directory is a file: " + directoryPath.toAbsolutePath());
+                } catch (NoSuchFileException e) {
+                    System.out.println("Directory not found: " + directoryPath.toAbsolutePath());
+                } catch (IOException e) {
+                    System.out.println("Something is wrong with the directory: " + directoryPath.toAbsolutePath());
+                    // Using print stack trace because can't seem to extract a useful error message otherwise
+                    e.printStackTrace();
```

Comment:
hmmm, worst case should be able to use https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html#getMessage()

Reply:
```
e.toString()
e.getMessage()
e.getLocalizedMessage()
```
results in
```
java.nio.file.NotDirectoryException: /tmp/docker_images10899984823268123628/fakeImage
/tmp/docker_images10899984823268123628/fakeImage
/tmp/docker_images10899984823268123628/fakeImage
```
I suppose the first one is alright